lib/betagouvbot/notification_rule.rb

```diff
@@ -6,12 +6,16 @@ class NotificationRule
     attr_reader :horizon, :mail_file, :recipients
 
     class << self
+      def all
+        [in_three_weeks, in_two_weeks, in_one_day, one_day_after]
+      end
+
       def horizons
         all.map(&:horizon)
       end
 
-      def all
-        [in_three_weeks, in_two_weeks, in_one_day, one_day_after]
+      def find(horizon:)
```

Comment:
I'm not sure I understand the syntax here. Typo? If not, I could use an explanation.